lbrynet/conf.py

```diff
@@ -43,25 +44,49 @@
     '.yml': yaml.safe_dump
 }
 
-if sys.platform.startswith('darwin'):
+if 'darwin' in sys.platform:
     platform = DARWIN
-    default_download_directory = os.path.join(os.path.expanduser('~'), 'Downloads')
+    default_download_dir = os.path.expanduser('~/Downloads')
     default_data_dir = user_data_dir('LBRY')
-    default_lbryum_dir = os.path.join(os.path.expanduser('~'), '.lbryum')
-elif sys.platform.startswith('win'):
+    default_lbryum_dir = os.path.expanduser('~/.lbryum')
+elif 'nt' in sys.platform:
     platform = WINDOWS
     from lbrynet.winhelpers.knownpaths import get_path, FOLDERID, UserHandle
 
-    default_download_directory = get_path(FOLDERID.Downloads, UserHandle.current)
-    default_data_dir = os.path.join(
-        get_path(FOLDERID.RoamingAppData, UserHandle.current), 'lbrynet')
-    default_lbryum_dir = os.path.join(
-        get_path(FOLDERID.RoamingAppData, UserHandle.current), 'lbryum')
+    default_download_dir = get_path(FOLDERID.Downloads, UserHandle.current)
+
+    # This checks whether the folders are still in their old locations
+    if os.path.isdir(user_data_dir('lbryum', 'lbry', roaming=True)) and \
+       os.path.isdir(user_data_dir('lbrynet', 'lbry', roaming=True)):
+        default_data_dir = user_data_dir('lbrynet', 'lbry', roaming=True)
+        default_lbryum_dir = user_data_dir('lbryum', 'lbry', roaming=True)
+    else:
+        default_data_dir = user_data_dir('lbrynet', 'LBRY')
```

Comment:
I guess it depends, Microsoft tends to love PascalCasing. If the program's name is LBRY, I guess the standard is the name the folder LBRY as well